### PR TITLE
Add new Github actions

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?

This PR adds 2 new Github actions:
- on pull request validate all schemas
- on release to zip up all schemas and add to release

### How to review

Github actions run successfully and do what they're designed to do

### CODE REVIEW

**Comment by rmccar** [comment]
[ons-questionnaire-schema Github Actions (2 SP)](https://trello.com/c/oL3M4o8X/4960-ons-questionnaire-schema-github-actions-2-sp)

**Comment by MebinAbraham** [comment]
For testing purposes, I am going to make a temp release on this repo against this branch. I don't think using a fork is necessary since this repo is not being used yet.

**Review by MebinAbraham — APPROVED** [review]
LGTM.

https://github.com/ONSdigital/eq-questionnaire-schemas/releases/tag/temp-v1.12.0 can be deleted when this is merged @rmccar

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: PR
+
+on: pull_request
```
Comment (MebinAbraham) [inline .github/workflows/pull_request.yml:3]: I think you want to set the branches the PR can be against, so it is only run against `main`?

**Comment by rmccar** [comment]
> Once this is merged, need to enable the status check in GitHub branch settings: image

Cool yeah will do. Thought I'd added this but maybe I didn't click save

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**